[#107] Add Unity 6.6 BuildReport Summary fields and ContentSummary tables (#108)
Unity-Technologies/UnityDataTools: Documentation/buildreport.md

Author: ```suggestion
For comprehensive analysis, run `analyze` on both the build output **and** the matching build report file. For Player and AssetBundle builds use a clean build, to ensure PackedAssets information is fully populated. 
```

* **BuildReport** - The primary object containing build inputs and results
* **PackedAssets** - Describes the contents of each SerializedFile, .resS, or .resource file, including type, size, and source asset for each object or resource blob, enabling object-level analysis

> * **ContentSummary** (class id 330615474) - A high-level content summary added in Unity 6.6: cross-build totals plus per-type and per-source-asset breakdowns. Present in Unity 6.6+ Player, AssetBundle, and ContentDirectory reports (absent from older reports and scripts-only builds).

Author: ```suggestion
* **ContentSummary** - A high-level content summary added in Unity 6.6: cross-build totals plus per-type and per-source-asset breakdowns. Present in Unity 6.6+ Player, AssetBundle, and content directory reports (absent from older reports and scripts-only builds).
```

> **Note:** PackedAssets information is not currently written for scenes in the build.

Author: ```suggestion
**Note:** Prior to Unity 6.6, PackedAssets information was not written for scenes in the build.
```

| `build_report_content_summary` | table | Cross-build content totals (sizes and counts). See [ContentSummary](https://docs.unity3d.com/6000.6/Documentation/ScriptReference/Build.Reporting.ContentSummary.html) |
| `build_report_content_type_stats` | table | Per Unity object type: size, object count, resource count |
| `build_report_content_asset_stats` | table | Per source asset: GUID, path, size, object count, resource count |
| `build_report_files_view` | view | All files from all build reports |
| `build_report_packed_assets_view` | view | All PackedAssets with their BuildReport, Archive, and SerializedFile |
| `build_report_packed_asset_contents_view` | view | All objects and resources tracked in build reports |
| `build_report_content_summary_view` | view | Cross-build content totals with their BuildReport |
| `build_report_content_type_stats_view` | view | Per-type stats with the type name and their BuildReport |

The `build_reports` table contains primary build information. Additional tables store detailed content data. Views simplify queries by automatically joining tables, especially when working with multiple build reports.

> These tables and views are created on demand, so a database analyzed without any build report does not contain them. The `build_reports`, `build_report_files`, `build_report_archive_contents` group and `build_report_files_view` are created when the first BuildReport object is analyzed; the `build_report_packed_*` tables and views are created when the first PackedAssets object is analyzed (a build report with no PackedAssets objects, such as some scene-only builds, will not have them); the `build_report_content_*` tables and views are created when the first ContentSummary object is analyzed (Unity 6.6+ only).

Author: ```suggestion
These tables and views are created on demand, so a database analyzed without any build report does not contain them. The `build_reports`, `build_report_files`, `build_report_archive_contents` group and `build_report_files_view` are created when the first BuildReport object is analyzed; the `build_report_packed_*` tables and views are created when the first PackedAssets object is analyzed (a build report with no PackedAssets objects, such as some scene-only builds, will not have them); the `build_report_content_*` tables and views are created if the file contains a ContentSummary object (Unity 6.6+ only).
```

> The new-in-6.6 `build_reports` columns (`build_name`, `build_content_options`, `build_session_guid`, `build_manifest_hash`, `build_profile_path`, `build_profile_guid`, `data_path`) are left NULL when analyzing reports from older Unity versions.

Author: ```suggestion
```
